test/ad/distributions.jl

```diff
@@ -550,4 +550,10 @@
             )
         end
     end
+
+    @testset "Turing issue 1385" begin
+        dist = arraydist(truncated.(Laplace.(0, [1, 2]), -10.0, 70.0))
+        x = ReverseDiff.track(rand(dist))
+        @test typeof(bijector(dist)(x)) <: ReverseDiff.TrackedArray
+    end
```

Comment:
Maybe it would also be good to really test the underlying issue by testing the output of `maporbroadcast` for ReverseDiff (and making sure, of course, that it fails on master) and not use any bijectors at all here.

Reply:
BTW it seems the test fails.